rktl_perception/src/focus_vis_node.cpp

```diff
@@ -0,0 +1,51 @@
+/* Node to detect the ball's position on the field.
+ * License:
+ *   BSD 3-Clause License
+ *   Copyright (c) 2020, Autonomous Robotics Club of Purdue (Purdue ARC)
+ *   All rights reserved.
+ */
+
+#include "rktl_perception/focus_vis_node.h"
+
+FocusNode::FocusNode() :
+    nh{},
+    pnh{"~"},
+    image_transport{nh},
+    imgPub{image_transport.advertise("edge_dectection", 1)},
+    camera_subscriber{image_transport.subscribeCamera("image_rect_color", 10, &FocusNode::FocusCallback, this)}
+
+    {}
+
+void FocusNode::FocusCallback(const sensor_msgs::ImageConstPtr& msg, const sensor_msgs::CameraInfoConstPtr& info) {
+    try {
+        /* Convert the ROS message into a cv_ptr & dereferencing the pointer to an OpenCV Mat */
+        cv_bridge::CvImagePtr cv_ptr = cv_bridge::toCvCopy(msg, "bgr8");
+        cv::Mat current_frame = cv_ptr->image;
+        cv::Mat frame_gray, frame_blurred, frame_edge;
+        
+        #if false /* shouldn't be necessary on the new computers */
```

Comment:
Why is this here? I think we can just remove it

Reply:
I presume you're talking about the resizing, which is gone. if you're talking about the `cv::Mat` stuff, that has to stay.

Reply:
I meant the \#if false stuff

Reply:
gone & pushed